packages/MainUI/screens/Table/Dynamic.tsx

```diff
@@ -1,8 +1,10 @@
+'use client';
```

Comment:
is this really needed? I think the layout has the directive so anything nested under it, will be already client sided